filebeat/harvester/log.go

```diff
@@ -105,6 +106,11 @@ func (h *Harvester) Harvest() {
 		if !h.sendEvent(event) {
 			return
 		}
+
+		if h.Config.CloseTTL > 0 && time.Since(h.startTime) > h.Config.CloseTTL {
```

Comment:
config validation mandates CloseTTL can not be 0 (due to `min=0,nonzero`). This on purpose?